public/src/app.js

```diff
@@ -770,12 +785,11 @@ app.cacheBuster = null;
 					app.alertSuccess('[[notifications:email-confirm-sent]]');
 				});
 			};
-
-			app.alert(msg);
 		} else if (!app.user['email:confirmed'] && app.user.isEmailConfirmSent) {
 			msg.message = '[[error:email-not-confirmed-email-sent]]';
-			app.alert(msg);
 		}
+
+		app.alert(msg);
```

Comment:
This will show an empty toaster if user has confirmed their email but doesn't have posting privilege in the category?

Reply:
Oh good catch.